themes/devious-diamonds.omp.yaml

```diff
@@ -0,0 +1,241 @@
+      - type: kubectl
+        style: powerline
+        powerline_symbol: 
+        background: lightYellow
+        foreground: black
+        template: " ﴱ {{.Context}}{{if .Namespace}} :: {{.Namespace}}{{end}} "
+      - type: node
+        style: powerline
+        powerline_symbol: 
+        background: lightGreen
+        foreground: black
+        properties:
+          fetch_version: true
+        template: "  {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} "
+      - type: php
+        style: powerline
+        powerline_symbol: 
+        background: lightCyan
+        foreground: black
+        template: "  {{ .Full }} "
+      - type: project
+        style: powerline
+        powerline_symbol: 
+        background: lightYellow
+        foreground: black
+        template: " {{ if .Error }}{{ .Error }}{{ else }}{{ if .Version }} {{.Version}}{{ end }} {{ if .Name }}{{ .Name }}{{ end }}{{ end }} "
+      - type: python
+        style: powerline
+        powerline_symbol: 
+        background: lightYellow
+        foreground: black
+        properties:
+          display_mode: files
+          fetch_virtual_env: false
+        template: "  {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} "
+      - type: ruby
+        style: powerline
+        powerline_symbol: 
+        background: red
+        foreground: black
+        properties:
+          display_mode: files
+          fetch_version: true
+        template: "  {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} "
+      - type: time
+        style: powerline
+        powerline_symbol: 
+        background: yellow
+        foreground: black
+        properties:
+          time_format: Mon | 15:04:05
+        template: " {{ .CurrentDate | date .Format }} "
+      - type: executiontime
+        style: diamond
+        trailing_diamond: 
+        background: cyan
+        foreground: black
+        properties:
+          style: austin
+          threshold: 0
+        template: " {{ .FormattedMs }} "
+  - type: prompt
+    alignment: right
+    segments:
+      - type: git
+        style: diamond
+        leading_diamond: 
+        trailing_diamond: 
+        background: magenta
+        foreground: black
+        properties:
+          branch_icon: " "
+          fetch_stash_count: true
+          fetch_status: true
+          fetch_upstream_icon: true
+          fetch_worktree_count: true
+        template: "{{ .UpstreamIcon }}{{ .HEAD }}{{ .BranchStatus }}{{ if .Working.Changed }}  {{ .Working.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Staging.Changed }}  {{ .Staging.String }}{{ end }}{{ if gt .StashCount 0 }}  {{ .StashCount }}{{ end }}"
+  - type: prompt
+    alignment: left
+    newline: true
+    segments:
+      - type: text
+        style: plain
+        foreground: cyan
+        template: " ╚"
```

Comment:
also, these should be the actual `\u` notations

Reply:
Oh no. I used an online converter and it spit the encoded chars out. I guess I'll have to redo it manually? Bleh.

What is the reasoning behind using `\u` notation? It's certainly easier to visually edit when they appear as the intended symbols.

Reply:
@JanDeDobbeleer `oh-my-posh config export --output ~/.mytheme.omp.yaml` exports the encoded characters. If it was set up to detect characters in the [Private Use Areas](https://en.wikipedia.org/wiki/Private_Use_Areas) and change their encoding, then that would give us a tool to do this automatically. I know I could make this work in Python, but I'm not confident enough to mess with your Go code on this one.

https://stackoverflow.com/questions/29161300/how-can-i-get-the-unicode-value-of-a-character-in-go